src/overlays/actors/ovl_En_Sob1/z_en_sob1.c

```diff
@@ -262,59 +269,76 @@ u16 EnSob1_GetWelcome(EnSob1* this, PlayState* play) {
             case PLAYER_FORM_HUMAN:
                 if (CHECK_WEEKEVENTREG(WEEKEVENTREG_57_10)) {
                     return 0x12CF;
+                } else {
+                    SET_WEEKEVENTREG(WEEKEVENTREG_57_10);
+                    return 0x12CE;
                 }
-                SET_WEEKEVENTREG(WEEKEVENTREG_57_10);
-                return 0x12CE;
+                break;
```

Comment:
Do these actually need breaks? Not convinced it's necessary when both paths clearly have a return.